games-engines/dragengine/dragengine-1.7.1.recipe

```diff
@@ -14,10 +14,10 @@ HOMEPAGE="https://dragondreams.ch
 COPYRIGHT="2020-2021 DragonDreams"
 LICENSE="GNU GPL v3
 	GNU LGPL v3"
-REVISION="3"
-HAIKU_PATCH="-haiku3"
+REVISION="1"
+HAIKU_PATCH=""
```

Comment:
Is it still needed?

Reply:
Not in this recipe. I kept the value in case I need to make a patch version in an upcoming recipe. If though this is a problem I can remove it.

Reply:
No other recipe use it so I think it would be cleaner to drop it.